handsontable/src/translations/indexMapper.js

```diff
@@ -437,6 +437,26 @@ export class IndexMapper {
     this.runLocalHooks('init');
   }
 
+  /**
+   * Trim/extend the mappers to fit the desired length.
+   *
+   * @param {number} length New mapper length.
+   */
+  fitToLength(length) {
```

Comment:
I've seen that you cover the change with [the e2e test](https://github.com/handsontable/handsontable/pull/8972/files#diff-b44f35a1b74ccc2a1eee272e7e75e690555997a55c3685375eefbc4128489e63R607) but there is no unit test for that specific module. Can you add one? Probably there `handsontable/src/translations/__tests__/indexMapper.unit.js`.